src/ray/raylet/scheduling/cluster_resource_scheduler_test.cc

```diff
@@ -336,8 +336,7 @@ TEST_F(ClusterResourceSchedulerTest, SchedulingUpdateAvailableResourcesTest) {
     int64_t violations;
     bool is_infeasible;
     int64_t node_id = resource_scheduler.GetBestSchedulableNode(
-        resource_request, false, false, &violations, &is_infeasible);
-    ASSERT_TRUE(node_id != -1);
+        resource_request, false, false, false, &violations, &is_infeasible);
```

Comment:
Can you add a test for a case where the policy should return a different node if grant_or_reject=true vs false?